src/Auth/Passwords/PasswordBrokerManager.php

```diff
@@ -19,7 +19,8 @@ protected function createTokenRepository(array $config)
             $this->app->make('registry')->getConnection($connection),
             $config['table'],
             $this->app['config']['app.key'],
-            $config['expire']
+            $config['expire'],
+            $config['throttle'] ?? 0
```

Comment:
It is 60 for all users today (default when not supplied). But this patch would set it to zero. Should we use `?? 60` instead perhaps?

Reply:
Yes, we can change it to default 60.